Makefile.in

```diff
@@ -907,6 +916,7 @@ stamps/build-musl-linux: $(MUSL_SRCDIR) $(MUSL_SRC_GIT) stamps/build-gcc-musl-st
 	+flock $(SYSROOT)/.lock $(MAKE) -C $(notdir $@) install
 	mkdir -p $(dir $@) && touch $@
 
+stamps/build-gcc-musl-stage2: ENABLED_LANGUAGES?="c,c++"
```

Comment:
Tangential to this specific PR but ...

I realise that this replicates what went before - i.e. that the Linux/Musl toolchain enables C/C++ only - but I'm now wondering if this was a mistake and this Linux toolchain should also enable Fortran by default? Or is there some reason why Fortran can't be enabled for the Linux/Musl RISC-V toolchain? Maybe others (e.g. @cmuellner, @kito-cheng, @aswaterman etc.) might know?

Reply:
My colleagues told me no one use fortran for bare-metal. That make sense.

Reply:
> My colleagues told me no one use fortran for bare-metal. That make sense.
> 
> At least I haven't use fortran.

Yes, but the Musl toolchain is a Linux toolchain - specifically Linux/Musl instead of Linux/Glibc. 

Perhaps the addition of Fortran to the Musl build should be a separate PR? I just noticed the seeming anomaly during the review of this one.